faq.md

```diff
@@ -0,0 +1,23 @@
+# Overview
+
+Contains [frequently asked questions](https://en.wikipedia.org/wiki/FAQ)
+related to Trifa Tox Client.
+
+
+# List of Content
+
+1. [What are the circles next to a contact?](#what-are-the-circles-next-to-a-contact)
+
+
+# What are the circles next to a contact?
+
+There are two circles next to a contact, see
+
+![Example Trifa Contact List](./doc/trifa-contact-list.png)
+
+* if `1` is
+    * green, it means that the contact is **online**
+    * red, it means that the contact is **offline**
+* if `2` is
+    * green, it means that the contact ??
+    * red, it means that the contact ??
```

Comment:
i guess it will need more / different screenshots.
the small circle can have 3 colors corresponding to the "away" status that you can select in the top menu of trifa.

and then there is a large color circle around the avatar image, which is:
grey -> offline
yellow -> online via TCP
green -> online via UDP

and the larger of the 2 circles can have a small inner circle, which can be:
yellow -> friend has PUSH notification feature
red -> friend has toxproxy (for offline messages) but it is offline 
green -> friend has toxproxy (for offline messages) and proxy is online

something like that, i hope i didnt miss anything ...

Reply:
Oh, i see. didn't know it's has so many meanings, nice.

Ideally, it could be
1. either self-explaning by some sort of icons? (have no idea yet, because it contains so many hints) 
2. reachable from UI

Regarding screen-shots..

Could you provide the screen-shots? Or maybe you know a way of getting them?

Android Emulator would be very time consuming for me to set up and i'm not even sure it supports my OS.

Reply:
i will make some screenshots

Reply:
Just updated the content, looks good enough to me.

Github markdown renderer skips the `span` colors, see
https://github.com/xliiv/ToxAndroidRefImpl/blob/add-faq/faq.md

.o.

So i can remove them, if we want.